### PR TITLE
New step 2

### PR DESCRIPTION
**Pull request by nsiddarthreddy**
This is what you wanted right?

### CODE REVIEW

File: levenshtein/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'Levenshtein'
+        db.create_table(u'levenshtein_levenshtein', (
+            (u'id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('text_1', self.gf('django.db.models.fields.TextField')()),
+            ('text_2', self.gf('django.db.models.fields.TextField')()),
+            ('levenshtein_value', self.gf('django.db.models.fields.IntegerField')(null=True, blank=True)),
```
Comment (diegobz) [inline levenshtein/migrations/0001_initial.py:16]: Tip: Your model already is called `Levenshtein`. No need to repeat that in the field names. `levenshtein_value` could be called `distance`, for example.

File: levenshtein/migrations/0002_auto__add_field_levenshtein_levenshtein_radio_info.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Levenshtein.levenshtein_radio_info'
+        db.add_column(u'levenshtein_levenshtein', 'levenshtein_radio_info',
```
Comment (diegobz) [inline levenshtein/migrations/0002_auto__add_field_levenshtein_levenshtein_radio_info.py:12]: Same comment as above applies here.

**Comment by diegobz** [comment]
Please also display the `ratio` for every entry in the HTML.

**Comment by nsiddarthreddy** [comment]
i made the changes you suggested , please take a look.

**Pull request merged**